apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -420,47 +451,57 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
 
   const googleBlocklyMixin = blocklyWrapper.BlockSvg.prototype.mixin;
-  blocklyWrapper.BlockSvg.prototype.mixin = function (
-    mixinObj,
-    opt_disableCheck
-  ) {
+  blocklyWrapper.BlockSvg.prototype.mixin = function (mixinObj) {
     googleBlocklyMixin.call(this, mixinObj, true);
   };
 
-  blocklyWrapper.BlockSvg.prototype.isDisabled = function () {
-    return this.disabled;
+  const extendedBlockSvg = blocklyWrapper.BlockSvg
+    .prototype as ExtendedBlockSvg;
+
+  extendedBlockSvg.isDisabled = function () {
+    return !this.isEnabled();
   };
 
-  blocklyWrapper.BlockSvg.prototype.getHexColour = function () {
+  extendedBlockSvg.getHexColour = function () {
```

Comment:
It looks like this one is never called. The comment below suggests it's expected that we wouldn't need it by now, but I'm not seeing evidence we were ever using calling it in the first place.

Reply:
added to follow-up ticket!